### DISCUSSION TITLE
TypeError: unsupported operand type(s) for +: 'frozenset' and 'list'

### DISCUSSION THREAD

**Discussion by statsconchris** [django-wiki/django-wiki#1257] [Bug] [answered]
While installing the newest version of django-wiki I encounter this error.

It appears in this file: `...../site-packages/wiki/conf/settings.py", line 60, in <module>`

Using `pip install --pre wiki`  doesn't fix the problem.

**Comment by oscarmcm** [comment]
Hi @statsconchris we've already fixed this via #1253 but we haven't released it yet.

**Comment by oscarmcm** [comment, marked as answer]
In the meantime you can install the latest version of Django-Wiki via GitHub with this:

`pip install git+https://github.com/django-wiki/django-wiki.git`
